File: back/APIParqueadero/App.Infrastructure/Database/Entities/Usuario.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace App.Infrastructure.Database.Entities
{
    public class Usuario
    {
        [Key]
        public int IdUsuario { get; set; }
        [Required]
        public string Nombre { get; set; }
        [Required]
        public string Apellido { get; set; }
        [Required]
        public DateTime FechaNacimiento { get; set; }
        [Required]
        public int IdTipoDoc { get; set; }
        [Required]
        public string Documento { get; set; }
        [Required]
        public string Contacto { get; set; }
        [Required]
        public string Correo { get; set; }
        [Required]
        public string Direccion { get; set; }
    }
}

File: front/Parqueadero/src/app/models/Usuario.ts
export class Usuario {

    /*IdUsuario : number = 0;
    Nombre : string = '';
    Apellido : string = '';
    FechaNacimiento : Date = new Date();
    IdTipoDoc : number = 0;
    Documento : string = '';
    Contacto : string = '';
    Correo : string = '';
    Direccion : string = '';*/
    
    private _IdUsuario : number = 0;
    public get IdUsuario() : number {
        return this._IdUsuario;
    }
    public set IdUsuario(v : number) {
        this._IdUsuario = v;
    }
    
    private _Nombre : string = '';
    public get Nombre() : string {
        return this._Nombre;
    }
    public set Nombre(v : string) {
        this._Nombre = v;
    }
    
    private _Apellido : string = '';
    public get Apellido() : string {
        return this._Apellido;
    }
    public set Apellido(v : string) {
        this._Apellido = v;
    }
    
    private _FechaNacimiento : Date = new Date();
    public get FechaNacimiento() : Date {
        return this._FechaNacimiento;
    }
    public set FechaNacimiento(v : Date) {
        this._FechaNacimiento = v;
    }
    
    private _IdTipoDoc : number = 0;
    public get IdTipoDoc() : number {
        return this._IdTipoDoc;
    }
    public set IdTipoDoc(v : number) {
        this._IdTipoDoc = v;
    }
    
    private _Documento : string = '';
    public get Documento() : string {
        return this._Documento;
    }
    public set Documento(v : string) {
        this._Documento = v;
    }
    
    private _Contacto : string = '';
    public get Contacto() : string {
        return this._Contacto;
    }
    public set Contacto(v : string) {
        this._Contacto = v;
    }
    
    private _Correo : string = '';
    public get Correo() : string {
        return this._Correo;
    }
    public set Correo(v : string) {
        this._Correo = v;
    }
    
    private _Direccion : string = '';
    public get Direccion() : string {
        return this._Direccion;
    }
    public set Direccion(v : string) {
        this._Direccion = v;
    }

    
}